src/core/p5.Renderer2D.js

```diff
@@ -621,24 +621,19 @@ class Renderer2D extends p5.Renderer{
         return this;
       }
     }
-    const kappa = 0.5522847498,
-      // control point offset horizontal
-      ox = w / 2 * kappa,
-      // control point offset vertical
-      oy = h / 2 * kappa,
-      // x-end
-      xe = x + w,
-      // y-end
-      ye = y + h,
-      // x-middle
-      xm = x + w / 2,
-      ym = y + h / 2; // y-middle
+    const xm = x + w / 2, // x-middle
```

Comment:
Thanks for doing this! 

Small point: maybe use centerX, centerY as constant names, instead of xm, ym? This is more consistent with radiusX, radiusY, and it's more self documenting. You could just eliminate the trailing comments.

Reply:
resolved it in https://github.com/processing/p5.js/pull/6499/commits/b73af847cc0c949630710fa162e1f95316cd3d63

Reply:
center X, center Y makes more sense 👍🏼